docs/modules/configuration/pages/tracing.adoc

```diff
@@ -2,12 +2,12 @@ include::ROOT:partial$attributes.adoc[]
 
 = Tracing block
 
-IMPORTANT: The `tracing` block is deprecated and will be removed in Cerbos 0.33.0. Refer to xref:observability.adoc#traces[observability configuration] for information about configuring traces.
+IMPORTANT: The `tracing` block was deprecated in Cerbos 0.32.0 and removed in Cerbos 0.33.0. Refer to xref:observability.adoc#traces[observability configuration] for information about configuring traces.
```

Comment:
This page is still in the nav as "Tracing (deprecated)", should we maybe remove it from the nav altogether now (leaving this page for Google etc)?